README.md

```diff
@@ -73,6 +73,13 @@ Projects using Trust Wallet Core.  Add yours too!
 | [Alice](https://www.alicedapp.com/)
 | [Frontier](https://frontier.xyz/)
 
+# Community
+
+There are a few community-maintained projects that extend Wallet Core to some additional platforms and languages. Note this is not an endorsement, please do your own research before using them:
```

Comment:
Can you remove "a few"?

Reply:
take a look again? I replaced with `some additional`